mlrun/db/httpdb.py

```diff
@@ -5345,15 +5351,18 @@ def revoke_secret_token(self, token_name: str) -> None:
     def _store_secret_tokens(
         self,
         secret_tokens: list[mlrun.common.schemas.SecretToken],
+        force: bool,
         error: str,
     ) -> mlrun.common.schemas.StoreSecretTokensResponse:
         body = [token.dict() for token in secret_tokens]
+        params = {"force": str(force).lower()}  # send as query param
```

Comment:
as we do in all other places in httpdb
```suggestion
        params = {"force": bool2str(force)}  
```

Reply:
Just leaving it as a bool also works